stubs/Contracts/Pagination.stub

```diff
@@ -3,22 +3,39 @@
 namespace Illuminate\Contracts\Pagination;
 
 /**
- * @mixin \Illuminate\Support\Collection
- * @mixin \Illuminate\Pagination\Paginator
+ * @template TItem
+ *
+ * @mixin \Illuminate\Support\Collection<array-key, TItem>
+ * @mixin \Illuminate\Pagination\Paginator<TItem>
```

Comment:
Maybe put these mixin tags into the concrete classes. `AbstractCursorPaginator` and `AbstractPaginator`

Same for the other interfaces below.

Reply:
✅